src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -521,6 +525,34 @@ export default Vue.extend({
       }
     },
 
+    fullscreenOverlay: function () {
+      const v = this
+
+      if (this.player.isFullscreen()) {
+        this.player.ready(function () {
+          v.player.overlay({
+            overlays: [{
+              showBackground: false,
+              content: `<h1>${document.title}</h1>`,
```

Comment:
Using the document title is fine, however please remove the " - FreeTube" string that's at the end of the title so that only the title is shown in the overlay.

Reply:
Do you want me to get rid of the "- Freetube" postfix in the `document.title`? That would also affect the title of the application that [I added](https://github.com/FreeTubeApp/FreeTube/pull/739).

Or do you still want the postfix in the application title but not the overlay title?

Reply:
Let's keep the postfix in the document title, but remove it for the overlay.